src/pages/Router.tsx

```diff
@@ -0,0 +1,28 @@
+import { Routes, Route } from 'react-router-dom';
+import NotFound from './NotFound';
+import FriendsList from './FriendsList';
+import Navigator from '../components/list/Naviagator';
+import RoomList from './RoomList';
+import styled from 'styled-components';
+import Setting from './Setting';
+const Router = () => {
+    return (
+        <MainPage>
+            <Navigator />
+            <Routes>
+                <Route path="/" element={<FriendsList />} />
```

Comment:
```suggestion
                <Route index element={<FriendsList />} />
```
index prop를 사용하면 상위 라우트의 경로와 일치하지만, 그 이후에 경로가 주어지지 않았을 때 보여지는 라우트를 설정할 수 있습니다. path="/"와 동일한 역할이지만 이를 좀 더 명시적으로 표현하는 방법입니다.